geant4.sh

```diff
@@ -88,28 +67,11 @@ module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@
 # Dependencies
 module load BASE/1.0 ${XERCESC_REVISION:+xercesc/$XERCESC_REVISION-$XERCESC_REVISION}
 # Our environment
-set osname [uname sysname]
 set GEANT4_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
 setenv GEANT4_ROOT \$GEANT4_ROOT
-setenv G4INSTALL \$GEANT4_ROOT
-setenv G4INSTALL_DATA \$GEANT4_ROOT/share/
-setenv G4SYSTEM \$osname-g++
-
-setenv G4ABLADATA ${G4ABLADATA:-not-defined}
-setenv G4ENSDFSTATEDATA ${G4ENSDFSTATEDATA:-not-defined}
-setenv G4INCLDATA ${G4INCLDATA:-not-defined}
-setenv G4LEDATA ${G4LEDATA:-not-defined}
-setenv G4LEVELGAMMADATA ${G4LEVELGAMMADATA:-not-defined}
-setenv G4NEUTRONHPDATA ${G4NEUTRONHPDATA:-not-defined}
-setenv G4NEUTRONXSDATA ${G4NEUTRONXSDATA:-not-defined}
-setenv G4PARTICLEXSDATA ${G4PARTICLEXSDATA:-not-defined}
-setenv G4PIIDATA ${G4PIIDATA:-not-defined}
-setenv G4RADIOACTIVEDATA  ${G4RADIOACTIVEDATA:-not-defined}
-setenv G4REALSURFACEDATA ${G4REALSURFACEDATA:-not-defined}
-setenv G4SAIDXSDATA ${G4SAIDXSDATA:-not-defined}
-set G4BASE \$GEANT4_ROOT
 prepend-path PATH \$GEANT4_ROOT/bin
-prepend-path ROOT_INCLUDE_PATH \$GEANT4_ROOT/include/Geant4
-prepend-path ROOT_INCLUDE_PATH \$GEANT4_ROOT/include
```

Comment:
just to understand: Which module file is now setting these variables (or who is setting these things when we load Geant4 on the GRID?)

Reply:
Ah I see. You modify the modulefile in a post-step.

Reply:
The data paths are added in the next step.
Geant4 includes need not to be added in the ROOT_INCLUDE_PATH as we do not generate dictionaries from Geant4 classes.